aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -403,6 +404,8 @@ func TestAccAWSFsxLustreFileSystem_DeploymentTypePersistent1(t *testing.T) {
 					// per_unit_storage_throughput is only available with deployment_type=PERSISTENT_1, so we test both here.
 					resource.TestCheckResourceAttr(resourceName, "per_unit_storage_throughput", "50"),
 					resource.TestCheckResourceAttr(resourceName, "deployment_type", fsx.LustreDeploymentTypePersistent1),
+					resource.TestMatchResourceAttr(resourceName, "mount_name", regexp.MustCompile(`^[a-z0-9]+fbmv$`)),
```

Comment:
The mount names always have the suffix `fbmv`, although that isn't documented publicly by AWS, so I don't know if we can consider that stable.

The other options are to just check that it's a lowercase alphanumeric string, or to check that it's *not* `fsx` which is the default mount name for `SCRATCH_1` deployment types.

Reply:
i ran the tests and the mount name didnt seem to be ending with `fbmv`.
```
    testing.go:684: Step 0 error: Check failed: Check 4/4 error: aws_fsx_lustre_file_system.test: Attribute 'mount_name' didn't match "^[a-z0-9]+fbmv$", got "mlvg5bmv"

```
it doesn't seem to have any logic, according to docs:

```
For the SCRATCH_1 deployment type, this value is always "fsx". For SCRATCH_2 and PERSISTENT_1 deployment types, this value is a string that is unique  within an AWS Region.
```

Lets change this to be not equal `fsx` as you proposed. i dont see any other way except just checking the value is set with no other validation.

Reply:
@DrFaust92 I implemented it as "just checking the value is set with no other validation".

I took a peek around the repository, and found the `resource.TestCheckResourceAttrSet` used in a bunch of places, which seems expressly for this purpose (a resource that we don't know the value ahead of time). I did not find any `resource.TestCheckResourceAttrDoesNotMatch` method in the `resource` go library.

So I figured it would be better to follow the existing convention, and not duplicate complex resource-retrieval logic from github.com/hashicorp/terraform-plugin-sdk/helper/resource. We're not doing any transformation on the `mount_name` retrieved from AWS, just passing it through, so ultimately I don't think an integration test that checks it's not equal to `fsx` would do much.

Let me know what you think. I can put together a "not `fsx`" integration test if you feel strongly.

Reply:
I think that just checking that the attribute is set is good enough